dashboard/views/data_document.py

```diff
@@ -2,14 +2,10 @@
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render, redirect, get_object_or_404
 from django.core.exceptions import ObjectDoesNotExist
-from djqscsv import render_to_csv_response
 from django.contrib import messages
 
-from dashboard.forms import *
-from dashboard.forms import ExtractedListPresenceTagForm
-# if this goes to 0, tests will fail because of what num form we search for
-from factotum.settings import EXTRA
-from dashboard.models import *
+from dashboard.forms import ExtractedListPresenceTagForm, create_detail_formset, DataDocumentForm, DocumentTypeForm
+from dashboard.models import DataDocument, ExtractedText
```

Comment:
```suggestion
from dashboard.models import DataDocument, ExtractedText, Script
```
We use the `Script` model on line 148 in `extracted_text_edit(request, pk)`. Tests fail if not imported.